docs/transactions/iti41.md

```diff
@@ -1,35 +1,228 @@
+
+The following rules must be observed when replacing a document :
+* [Patients](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-PAT) and their [representatives](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-REP) can only replace a document published by the same patient.
+	* representatives are not supported by the service yet.
+* [Healthcare professionals](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-HCP) (HCP) can only replace a document published by himself or by another HCP of same community of affiliation.
+    * Current implementation: an HCP can replace a document published by any other HCP. Further rules and possible implementations are under discussion.
+* [Document administrator](https://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html) may only replace a document published by an HCP of the same community of affiliation.
+	* Not implemented in PMP, to be discussed, for now a document admin can replace any document.
+* Replacing and replaced documents have to be of the same type (e.g. both MTP documents).
+* Replacing and replaced documents have to refer to the same patient.
+* Only documents at the end of a treatment chain can be replaced.
+* Replaced document must exist in the repository.
+* A replacement document must be linked only to elements of the same medication chain. In case a PRE document is replaced, it may be linked only to elements of the medication chains of the referenced MTP. 
+* [Document administrator](https://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html) can replace any document regardless of who published it.
 
 ## Publishing a PMP-APPC document
+This section details the rules applicable to metadata when publishing [APPC documents](https://www.ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_Suppl_APPC.pdf).
+
+### Rules for APPC:
+* Only  [Patients](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-PAT) and their [representatives](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-REP) or policy administrators can publish a new APPC document.
+	* representatives are not supported by the service yet.
+*  Only [Patients](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-PAT) and their [representatives](http://fhir.ch/ig/ch-epr-term/2.0.9/CodeSystem-2.16.756.5.30.1.127.3.10.6.html#2.16.756.5.30.1.127.3.10.6-REP) can replace their own APPC documents (if it exists).
+	* representatives are not supported by the service yet.
+* Policy administrators of the reference community of the patient can replace any existing APPC document.
+* No APPC for the specified patient exists in the system, otherwise the document is refused.
```

Comment:
Unless publishing a replacement.
Furthermore, as for CH-EMED-EPR, an APPC document to be replaced must:

- be approved (as before, this is always the case for documents in the PMP repository)
- not have `deletionStatus` = `deletionRequested`